dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report.repo.ts

```diff
@@ -174,15 +175,21 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
    * @returns The utilisation report with attached fee records
    */
   async findOneByIdWithFeeRecordsFilteredByStatus(reportId: number, feeRecordStatuses: FeeRecordStatus[]): Promise<UtilisationReportEntity | null> {
-    return await UtilisationReportRepo.findOne({
+    const report = await this.findOne({
+      where: { id: reportId },
+    });
+
+    if (!report) {
+      return null;
+    }
+
+    report.feeRecords = await FeeRecordRepo.find({
```

Comment:
I see that this works but it feels a little suboptimal as we have to be careful everywhere. I wonder if setting the entity to default to an empty array would work and is a better solution. e.g. I haven't tried it but I'm hoping this could work:
```
@OneToMany(() => FeeRecordEntity, (feeRecord) => feeRecord.report, {
   cascade: ['insert', 'update'],
})
feeRecords: FeeRecordEntity[] = [];
```
Would you mind trying that if you have't already?